bigchaindb/elections/vote.py

```diff
@@ -39,20 +41,20 @@ def validate(self, bigchain, current_transactions=[]):
     @classmethod
     def generate(cls, inputs, recipients, election_id, metadata=None):
         (inputs, outputs) = cls.validate_transfer(inputs, recipients, election_id, metadata)
-        election_vote = cls(cls.VALIDATOR_ELECTION_VOTE, {'id': election_id}, inputs, outputs, metadata)
+        election_vote = cls(cls.OPERATION, {'id': election_id}, inputs, outputs, metadata)
         cls.validate_schema(election_vote.to_dict(), skip_id=True)
```

Comment:
We can also remove `skip_id` from here.

Reply:
why?

Reply:
Because `Vote` is always a `TRANSFER`, and we always validate how identifiers are built for transfers.